spec/models/form_spec.rb

```diff
@@ -633,6 +633,7 @@
         payment_link_status: :optional,
         receive_csv_status: :optional,
         support_contact_details_status: :completed,
+        welsh_language_status: :optional,
```

Comment:
should `welsh_language_status` show up in this list even when the feature flag is not enabled for this form?

Is it that it doesn't get added to the tasklist and it's optional so it doesn't matter if it's always in the list?

Reply:
I've added a check against the FeatureFlag